[paraview] build failure

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
vcpkg install paraview

find_package(ParaView)
```


### Failure logs

CMake Error at build/vcpkg_installed/x64-windows/share/paraview/ParaView-targets.cmake:1116 (message):
  The imported target "ParaView::WrapClientServer" references the file

     "R:/ttk/build/vcpkg_installed/x64-windows/tools/paraview/vtkWrapClientServer-pv5.11d.exe"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "R:/ttk/build/vcpkg_installed/x64-windows/share/paraview/ParaView-targets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  build/vcpkg_installed/x64-windows/share/paraview/paraview-config.cmake:86 (include)
  R:/vcpkg/scripts/buildsystems/vcpkg.cmake:855 (_find_package)
  CMake/config.cmake:50 (find_package)
  CMakeLists.txt:12 (include)

### Additional context

in ParaViewClients-targets-debug.cmake:
```
# Import target "ParaView::WrapClientServer" for configuration "Debug"
set_property(TARGET ParaView::WrapClientServer APPEND PROPERTY IMPORTED_CONFIGURATIONS DEBUG)
set_target_properties(ParaView::WrapClientServer PROPERTIES
  IMPORTED_LOCATION_DEBUG "${_IMPORT_PREFIX}/tools/paraview/vtkWrapClientServer-pv5.11d.exe"
  )
```
Note the 'd' at the end

in https://github.com/microsoft/vcpkg/blob/8568364fbeb414db3fa5b5bfc436e57b4368c55f/ports/paraview/portfile.cmake#L152-L155
that debug exe is removed

same probably goes for: 
smTestDriver, vtkProcessXML

--- a/ports/paraview/portfile.cmake
+++ b/ports/paraview/portfile.cmake
@@ -197,6 +197,14 @@ if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
 endif()
 
+file(GLOB FILES "${CURRENT_PACKAGES_DIR}/share/${PORT}/*.cmake")
+
+foreach(FILE ${FILES})
+    file(READ "${FILE}" file_contents)
+    STRING(REPLACE "pv5.11d.exe" "pv5.11.exe" file_contents "${file_contents}")
+    file(WRITE "${FILE}" "${file_contents}")
+endforeach()   
+ 
 # The plugins also work without these files
 file(REMOVE "${CURRENT_PACKAGES_DIR}/Applications/paraview.app/Contents/Resources/paraview.conf")
 file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/Applications/paraview.app/Contents/Resources/paraview.conf")

--- a/ports/paraview/portfile.cmake
+++ b/ports/paraview/portfile.cmake
@@ -197,13 +197,10 @@ if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
 endif()
 
-file(GLOB FILES "${CURRENT_PACKAGES_DIR}/share/${PORT}/*.cmake")
-
-foreach(FILE ${FILES})
-    file(READ "${FILE}" file_contents)
-    STRING(REPLACE "pv5.11d.exe" "pv5.11.exe" file_contents "${file_contents}")
-    file(WRITE "${FILE}" "${file_contents}")
-endforeach()   
+file(GLOB cmake_files "${CURRENT_PACKAGES_DIR}/share/${PORT}/*.cmake")
+foreach(file IN LISTS cmake_files)
+    vcpkg_replace_string("${file}" "pv5.11d.exe" "pv5.11.exe")
+endforeach() 
  
 # The plugins also work without these files
 file(REMOVE "${CURRENT_PACKAGES_DIR}/Applications/paraview.app/Contents/Resources/paraview.conf")

--- a/ports/paraview/vcpkg.json
+++ b/ports/paraview/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "paraview",
   "version": "5.11.0",
-  "port-version": 3,
+  "port-version": 4,
   "description": "VTK-based Data Analysis and Visualization Application",
   "homepage": "https://www.paraview.org/",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6258,7 +6258,7 @@
     },
     "paraview": {
       "baseline": "5.11.0",
-      "port-version": 3
+      "port-version": 4
     },
     "parmetis": {
       "baseline": "2022-07-27",

--- a/versions/p-/paraview.json
+++ b/versions/p-/paraview.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f9b120802ef7b14447d29739db0769b8a57ac9ad",
+      "git-tree": "2aa3d8185a319a76c7961608ecce8b2f03c08cd7",
       "version": "5.11.0",
       "port-version": 4
     },

--- a/versions/p-/paraview.json
+++ b/versions/p-/paraview.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f9b120802ef7b14447d29739db0769b8a57ac9ad",
+      "version": "5.11.0",
+      "port-version": 4
+    },
+    {
       "git-tree": "ef783841125e47d511ecc6d0108e7b1ecaef1469",
       "version": "5.11.0",
       "port-version": 3

